### PR TITLE
Feature/13 grid view

### LINKED ISSUE #13: Search page - grid view

### PR DESCRIPTION
**Pull request by YoYuUm**
Closes #13

### CODE REVIEW

**Comment by lraivio** [comment]
I think now that we only have school pages the whole item in the grid view should be clickable and a link to the school page. Now the hover effect coloring the whole item makes me want to click the item from where ever and not just the name or the picture :)

File: app/components/SearchResultsGrid.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import _ from 'lodash';
+import React from 'react';
+import {Link} from 'react-router';
+import {processBasicInfoRow, getImageUrl} from '../core/utils';
+import InfoRow from './InfoRow';
+
+
+function getInfoRow(row) {
+  delete row.boxContent;
```
Comment (lraivio) [inline app/components/SearchResultsGrid.js]: Why is this needed?

File: app/core/utils.js [hunk 2]
```diff
@@ -11,6 +16,60 @@ function getAddressArrayFromBuilding(building) {
...
+  if (!building || !building.photos || !building.photos.length) {
+    return defaultImageUrl;
+  }
+  const photo = _.find(building.photos, 'is_front');
+  return photo ? photo.url : building.photos[0].url;
+}
+
 export default {
-  getAddressArrayFromBuilding: getAddressArrayFromBuilding
+  processBasicInfoRow,
```
Comment (lraivio) [inline app/core/utils.js]: This could be the last one so it's in alphabetical order.

File: app/styles/search-page.less [hunk 3]
```diff
@@ -55,4 +55,35 @@
...
+    .school-grid-row {
+      .row();
+      margin-top: 30px;
+      padding-bottom: 40px;
+      .school-grid-wrapper {
+        .make-column(4, 4);
+
+        .school-grid {
+          padding: 10px 2px 0px 2px;
+          border-bottom: 1px solid @border-main-color;
```
Comment (lraivio) [inline app/styles/search-page.less]: A slight transition for the background-color would be nice here:
`.transition(background-color 0.3s);`
Fancier animations and such can be added later.

File: app/components/SearchResultsGrid.js (hunk 1 above)
Comment (YoYuUm) [inline app/components/SearchResultsGrid.js]: To remove the infoBox ([ year -> year ]) from the row. I though about leaving the infoBox, but without it the contents in the page breath better, and it's closer to the original design.

Comment (lraivio) [inline app/components/SearchResultsGrid.js]: :+1:

**Comment by lraivio** [comment]
Awesome job! :+1:
